lib/kitchen/provisioner/chef_base.rb

```diff
@@ -131,8 +131,8 @@ def install_options
           :omnibus_url => config[:chef_omnibus_url],
           :project => project.nil? ? nil : project[1],
           :install_flags => config[:chef_omnibus_install_options],
-          :use_sudo => config[:sudo],
-          :sudo_command => config[:sudo_command]
+          :use_sudo => false,
```

Comment:
Instead of setting `false` and `nil` here, lets just leave them out - Mixlib::Install will use the defaults we're already specifying and then we have less code to migrate later.